docs/security-openid-connect.md

```diff
@@ -8,6 +8,10 @@ Apache Pulsar supports authenticating clients using [OpenID Connect](https://ope
 
 The source code for the OpenID Connect implementation is in the [pulsar-broker-auth-oidc](https://github.com/apache/pulsar/blob/master/pulsar-broker-auth-oidc/) submodule in the Apache Pulsar git repo.
 
+:::note
+Pulsar's OpenID Connect integration was introduced in Pulsar 3.0.0. As always, if you encounter any issues, please ask questions on Pulsar channels and open issues in GitHub.
```

Comment:
```suggestion
Pulsar's OpenID Connect integration is available from 3.0.0. 
```

> As always, if you encounter any issues, please ask questions on Pulsar channels and open issues in GitHub.

We do not need to add this because:
1) it's a common practice for developers
2) it's applicable for all features/improvements/... If we add this here, we need to add this everywhere

Reply:
> it's applicable for all features/improvements/... If we add this here, we need to add this everywhere

My intent is slightly different here than with other features. I agree that with open source projects, it is often implicit that we accept contributions from anyone interested in taking the time to give them. However, in this section, I wanted to convey that this is a new feature and there might be edge cases that are not covered in the docs. In some projects, they represent this by calling a feature "alpha", "beta", or "GA". We don't have those terms. Is there a different way I can word this to express that sentiment?

Reply:
How about this?

::: note

Pulsar's OpenID Connect integration is available from 3.0.0. Currently, this feature is in beta.

:::

Reply:
After further thought, I think your first suggestion is correct.